README.md

```diff
@@ -1,96 +1,50 @@
 These ligatures were custom created using [Glyphs](https://glyphsapp.com/).
 There are even italic versions of the ligatures.
 
-## New Version 2.2.5
-
-All new redesigned ligatures with better hinting. Updated font generation to support advanced OpenType features
-like those found in Fira Code. For example, the cursor now moves inside the ligature. It also handles repeating
-characters properly.
-
-<img src="./images/caret-position.gif" />
-
-### 🎉 New in Version 2.2
-
-With the awesome help from [Mark Skelton](https://github.com/mskelton), we now have the full set of ligatures for
-the following fonts. Thanks Mark, and thanks to all of you who have been patiently waiting for these ligatures to be completed.
-
-- Operator Mono SSm Light/Light Italic
-- Operator Mono SSm Book/Book Italic
-- Operator Mono SSm Medium/Medium Italic
-- Operator Mono SSm Bold/Bold Italic
-- Operator Mono Light/Light Italic
-- Operator Mono Book/Book Italic
-
-### 🐛 Fixed in Version 2.2.5
-
-Add Operator Mono Book/Book Italic ligatures.
-
-### 🐛 Fixed in Version 2.2.4
-
-Add some missing ligatures.
-
-### 🐛 Fixed in Version 2.2.3
-
-Updated Operator Mono SSm Book Italic ligatures.
-
-### 🐛 Fixed in Version 2.2.2
-
-Fixed glyph widths Operator Mono Light and Light Italic. These fonts accidentally were sized
-the same as the ScreenSmart versions and caused alignment issues. Also fixed a few ligatures
-that didn't have the correct weight.
-
-### 🔧 Customize the generated font
-
-In addition to the new ligatures, this version now allows you to customize what ligatures are added to a font.
-By default, all available ligatures will be added to the generated font. However, if you don't like a particular
-ligature, or would prefer to use an alternate glyph, you can create a `profiles.ini` file in the `./original` folder
-to configure how the font should be built.
-
-You can create one or more profiles. Each profile will have a set of directives. You can specify alternate glyphs for
-a given ligature. You can also prevent a ligature from being added.
+<img src="./images/operator-mono-lig.png" />
 
-Each profile will be listed with `[profile name]` (the first profile should be named default). The generated font will
-include the profile name. For example `[Go]` would generate the font: _Operator Mono Lig Go_. This way you can configure
-a different set of ligatures for each language in your favorite code editor.
+In addition to the graphic above, the following new ligatures are available:
 
-```ini
-# name of profile (one or more sections, first should be name default)
-[default]
-# glyph=altglyph
-greater_equal.liga=greater_equal.2.liga
+<img src="./images/new-ligatures.png"/>
 
-# do not include glyph (add ! prefix)
-!equal_equal.liga
+# Release notes: 
 
-# another optional section (will create a font named Operator Mono Lig Go)
-[Go]
-... add custom directives for this font ...
-```
+- 🎉 New Version 2.2.5
```

Comment:
I like the moving of release notes to a separate heading, but I think that each new version should be a new heading (so heading level 3 in this case) as it looks better and doesn't try to cram paragraphs into a simple bullet point list.

Reply:
Ha. v2.2.5 should have been 2.3.0 as I added a whole new set of fonts.

I do want to use SemVer (loosely as it's not really code), but to denote major/minor changes.

Not all versions will have more than a single line item change, so whatever structure you come up with, make sure it supports one or more items.

Reply:
I think SemVer can work for this? 

* Major changes are just major overhauls (if need be).
* Minor are essentially just adding ligatures
* Patch are just fixing any ligatures and changing the code related to compiling and stuff

Reply:
Maybe we should just add a changelog.md file?  We could link to it from the readme for visibility, but then we don't clutter up the readme.

Reply:
I'm OK with that. And simply update the README with the latest version and point them to the CHANGELOG for details.